packages/client/hmi-client/src/services/model.ts

```diff
@@ -247,3 +247,7 @@ export function stringToLatexExpression(expression: string): string {
 	latexExpression = latexExpression.replace(/([a-zA-Z0-9]+)\/([a-zA-Z0-9]+)/g, '\\frac{$1}{$2}');
 	return latexExpression;
 }
+
+export function getTimeUnits(model: Model): CalendarDateType {
+	return model?.semantics?.ode?.time?.units?.expression || '';
+}
```

Comment:
I feel like we should mark down a canonical place to house functions to set/get model attributes/properties. Right now they are kind of splattered over
- services/models, and
- model-representation/service

Do you have any preference since you created a large subset of these functions? We don't need to do the refactor now but would be good to get consensus and mark it down as a TODO.

Reply:
Ill mark a TODO...would be best to have these in one place.  I think services/models might be best